doc/source/whatsnew/v2.0.0.rst

```diff
@@ -164,6 +164,13 @@ Removal of prior version deprecations/changes
 - Removed deprecated :meth:`Index.is_type_compatible` (:issue:`42113`)
 - Removed deprecated :meth:`Index.is_mixed`, check ``index.inferred_type`` directly instead (:issue:`32922`)
 - Removed deprecated :meth:`Index.asi8` (:issue:`37877`)
+- Removed deprecated :meth:`DataFrame._AXIS_NUMBERS`, :meth:`DataFrame._AXIS_NAMES`, :meth:`Series._AXIS_NUMBERS`, :meth:`Series._AXIS_NAMES` (:issue:`33637)
```

Comment:
A tick is missing after the issue in this line

Reply:
thanks, fixed